guides/common/modules/proc_booting-virtual-machines.adoc

```diff
@@ -10,7 +10,7 @@ It has the following workflow:
 
 . Virtual machine starts
 . iPXE retrieves the network credentials using DHCP
-. iPXE retrieves the HTTP address using DHCP
+. iPXE retrieves the HTTP URL using DHCP
```

Comment:
This really is a single step, so perhaps they can be joined? Something like `iPXE retrieves the network credentials using DHCP, including HTTP URL`